src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
+import { Link } from 'react-router';
 
-function Dashboard() {
-  return <h2>Dashboard Two!</h2>;
-}
+import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
+
+import ApplyForBenefits from './apply-for-benefits/ApplyForBenefits';
+import ClaimsAndAppeals from './claims-and-appeals/ClaimsAndAppeals';
+import HealthCare from './health-care/HealthCare';
+
+const Dashboard = () => {
+  return (
+    <div className="vads-l-grid-container vads-u-padding-x--0">
+      <Breadcrumbs>
+        <a href="/" key="home">
+          Home
+        </a>
+        <Link to="/" key="dashboard">
+          My VA
+        </Link>
+      </Breadcrumbs>
+
+      <h1 id="dashboard-title" tabIndex="-1">
```

Comment:
These `h1` attributes are taken directly from [our current VA Dashboard](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/dashboard/containers/DashboardApp.jsx)

The `key` attributes on the `Links` were taken from [our current my VA Breadcrumbs](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/dashboard/containers/DashboardAppWrapper.jsx)